app/views/my-account/billing/index.pug

```diff
@@ -81,10 +81,10 @@ block body
                 label(for='input-keyword-search')= t('Keyword')
             .form-row
               .col.form-group.floating-label
-                input.form-control#input-start-date(type='date', value=ctx.query.startDate ? ctx.query.startDate : '', name='startDate', placeholder='YYYY/MM/DD')
+                input.form-control#input-start-date(type='date', value=ctx.query.start_date ? ctx.query.start_date : '', name='start_date', placeholder='YYYY/MM/DD')
```

Comment:
`placeholder` does not apply to `input` type of `date` via https://html.spec.whatwg.org/multipage/input.html#date-state-(type%3Ddate) fyi